### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by austinsonger** [NSWC-Crane/C-PAT#63] [General]
Would you be able to create some documentation on how to run this locally?

My company has to deal with maintaining a POAM  and I've been looking for open source solutions that can be deployed internally.

**Comment by austinsonger** [comment]
@crodriguez6497

**Comment by crodriguez6497** [comment]
@austinsonger This is definitely on the agenda, however we are still pre-beta release currently. I don't have an exact timeline on when we will have a containerized release ready but I hope to have some docs published within the next 60-90 days that will also include details on a local deployment from source code.

**Comment by austinsonger** [comment]
@crodriguez6497  Okay sounds good. If you need contribution help I can help with anything you need. Managing POAM is one of the biggest pains.
